.github/workflows/test-reproduction-run.yml

```diff
@@ -0,0 +1,30 @@
+name: Test Full Run
+
+on:
+  push:
+    branches: [master]
```

Comment:
The issue is here, this only runs on master. It should run on the same triggers as the other CI jobs.